vrecord_policy_uncompressed.xml

```diff
@@ -0,0 +1,65 @@
+        </policy>
+        <rule name="Is the video interlaced?" value="ScanType" tracktype="Video" operator="=">Interlaced</rule>
+        <rule name="Is the compression mode lossless?" value="Compression_Mode" tracktype="Video" operator="=">Lossless</rule>
+        <rule name="Is the colour description present?" value="colour_description_present" tracktype="Video" operator="=">Yes</rule>
+        <rule name="Are the transfer characteristics BT.709?" value="transfer_characteristics" tracktype="Video" operator="=">BT.709</rule>
+        <rule name="Are the matrix coefficients BT.601?" value="matrix_coefficients" tracktype="Video" operator="=">BT.601</rule>
+        <rule name="Is the audio stream on track 2?" value="StreamOrder" tracktype="Audio" operator="=">1</rule>
+        <rule name="Is the audio PCM?" value="Format" tracktype="Audio" operator="=">PCM</rule>
+        <rule name="Is the endianness Little?" value="Format_Settings_Endianness" tracktype="Audio" operator="=">Little</rule>
+        <rule name="Is the signedness Signed?" value="Format_Settings_Sign" tracktype="Audio" operator="=">Signed</rule>
+        <rule name="Is the PCM Codec ID in24?" value="CodecID" tracktype="Audio" operator="=">in24</rule>
+        <rule name="Is the bitrate mode CBR?" value="BitRate_Mode" tracktype="Audio" operator="=">CBR</rule>
+        <policy type="or" name="1 or 2 channels of audio?">
+            <rule name="1 channel of audio?" value="Channels" tracktype="Audio" operator="=">1</rule>
+            <rule name="2 channels of audio?" value="Channels" tracktype="Audio" operator="=">2</rule>
+        </policy>
+        <!--channel positions and channel layout?-->
+        <rule name="Is the sample rate 48kHz?" value="SamplingRate" tracktype="Audio" operator="=">48000</rule>
+        <rule name="Is the bit Depth 24?" value="BitDepth" tracktype="Audio" operator="=">24</rule>
+        <!--stream size proportion?-->
+        <policy type="or" name="Is it NTSC SD or PAL SD?">
+            <policy type="and" name="Is it NTSC?">
+                <rule name="Is the standard NTSC?" value="Standard" tracktype="Video" operator="=">NTSC</rule>
+                <rule name="Is the height 486?" value="Height" tracktype="Video" operator="=">486</rule>
+                <rule name="Is the width 720?" value="Width" tracktype="Video" operator="=">720</rule>
+                <rule name="Is the frame rate 29.970?" value="FrameRate" tracktype="Video" operator="=">29.970</rule>
+                <rule name="Is the pixel aspect ratio 0.900?" value="PixelAspectRatio" tracktype="Video" operator="=">0.900</rule>
+                <rule name="Is the scan order bottom field first?" value="ScanOrder" tracktype="Video" operator="=">BFF</rule>
+                <rule name="Are the colour primaries BT.601 NTSC?" value="colour_primaries" tracktype="Video" operator="=">BT.601 NTSC</rule>
+                </policy>
+                <policy type="and" name="Is it PAL?">
```

Comment:
I suggest to fix the alignment of this and the previous line, in order to visualise the structure.